tests/unit/cbprintf/main.c

```diff
@@ -371,7 +371,7 @@ static inline bool prf_check(const char *expected,
 	zassert_true(prf_check(expected, rv, __FILE__, __LINE__), \
 		     NULL)
 
-static void test_pct(void)
+void test_pct(void)
```

Comment:
I really don't like this commit either.  We should not have tests/examples that promote contamination of the global namespace with functions that are never referenced outside the translation unit.

If a function's commented out permanently, it should be removed.  If it's done during development, ignore the warnings.  If it's conditional on build options leave it in the list, and skip on entry when the triggering feature is disabled.